platform-hub-api/app/models/user.rb

```diff
@@ -14,4 +30,13 @@ def identity provider
     identities.find_by provider: provider
   end
 
+  def make_admin!
+    self.admin!
```

Comment:
Bit confusing I know; but this is a special way of directly setting an enum value on ActiveRecord object and have it persist immediately. I debated doing something similar to `revoke_admin!` below, but figured this would just about be okay.